build/Linux+BSD/portable/copy-libs

```diff
@@ -135,6 +251,9 @@ building_on_CentOS_6() {
   copyLib libpng12.so.0
   copyLib libudev.so.0
   copyLib liblcms.so.1
+
+  # Needed by Gentoo 2.2:
+  copyLib libpulse.so.0
```

Comment:
Thanks. I'm not as worried about "enthusiast" distributions like Arch and Gentoo as I am about "mainstream" distributions, but if someone reports that they need a library to make it work then I'm happy for it to be added.